src/main/java/logic/DoomLogic.java

```diff
@@ -71,19 +74,38 @@ public void onEvent(MessageCreateEvent event, Message message, User user, String
     }
 
     private EmbedBuilder handleCommand(String mode, String poolName, int count) {
-        switch (mode) {
-            case "add":
-                return DoomHandler.addDoom(poolName, count);
-            case "sub":
-                return DoomHandler.addDoom(poolName, count * -1);
-            case "select":
-                return DoomHandler.setActivePool(poolName);
-            case "set":
-                return DoomHandler.setDoom(poolName, count);
-            case "delete":
-                return DoomHandler.deletePool(poolName);
-            default:
-                return poolName.equals("") ? DoomHandler.generateDoomEmbed() : DoomHandler.generateDoomEmbed(poolName);
+        String actualPoolName = DoomHandler.findPool(poolName);
+        if (actualPoolName == null) {
+            if (mode == null || mode.isEmpty() || mode.equalsIgnoreCase("list")) {
+                return DoomHandler.generateDoomEmbed();
+            }
+            else {
+                return new EmbedBuilder()
+                        .setTitle("Error")
+                        .setDescription("No Doom Pool with Name ''**" + poolName + "**'' exists.");
```

Comment:
Can you print suggestions if there are any?

Reply:
I mean for the most part, if it's null it most likely didn't find any within 2 points of similarity , the "multiple" ones is more of an error catch than  an expected.